addons/website_sale_delivery/static/src/js/website_sale_delivery.js

```diff
@@ -100,6 +100,11 @@ publicWidget.registry.websiteSaleDelivery = publicWidget.Widget.extend({
         if (result.new_amount_total_raw !== undefined) {
```

Comment:
we do this code only if `result.new_amount_total_raw` !== undefined, if undefined is a possibility, shouldn't the new code only be called if the value is not undefined too?